python.qs

```diff
@@ -0,0 +1,191 @@
+var ComponentSelectionPage = null;
+
+var Dir = new function () {
+    this.toNativeSparator = function (path) {
+        if (systemInfo.productType === "windows")
+            return path.replace(/\//g, '\\');
+        return path;
+    }
+};
+
+
+function getPythonInfo() {
+
+}
+
+
+function Component() {
+    if (installer.isInstaller()) {
+        console.log(component.name);
+        component.loaded.connect(this, Component.prototype.installerLoaded);
+    }
+}
+
+Component.prototype.installerLoaded = function () {
+    var components = installer.components();
+    console.log(components);
+    components.forEach(component => {
+      var compName = component.name;
+      console.log(compName);
+    })
+
+    ComponentSelectionPage = gui.pageById(QInstaller.ComponentSelection);
+    // ComponentSelectionPage.NextButton.clicked.connect(this, Component.prototype.showOrHidePythonTools)
+    ComponentSelectionPage.left.connect(this, Component.prototype.pythonToolsWidgetEntered);
+
+
+  if (installer.addWizardPage(component, "PythonToolsWidget", QInstaller.ReadyForInstallation)) {
+    var widget = gui.pageWidgetByObjectName("DynamicPythonToolsWidget");
+    if (widget != null) {
+      widget.targetChooser.clicked.connect(this, Component.prototype.chooseTarget);
+      widget.targetDirectory.textChanged.connect(this, Component.prototype.targetChanged);
+
+      widget.windowTitle = "Python Tools";
+
+      var sep = ":";
+      if (systemInfo.productType === "windows") {
+        sep = ";"
+      }
+      var paths = installer.environmentVariable("PATH").split(sep);
+
+      console.log(paths);
+      var pythonpath = installer.findPath("python", paths);
```

Comment:
Initial lookup in PATH. On windows I probably need to add .exe or something, untested.